Dockerfile

```diff
@@ -6,12 +6,17 @@ COPY composer.lock /usr/src/app/
 
 RUN apk --update add git php-common php-xml php-dom php-ctype php-iconv php-json php-pcntl php-phar php-openssl php-opcache php-sockets curl && \
     curl -sS https://getcomposer.org/installer | php && \
-    /usr/src/app/composer.phar install && \
-    apk del build-base && rm -fr /usr/share/ri
+    /usr/src/app/composer.phar install
+RUN apk --update add build-base ruby-dev ruby ruby-bundler && gem install httparty --no-rdoc --no-ri
```

Comment:
I think one continuous line is better. I don't see us changing ruby often (it's not like touching the source code, or even the `Gemfile` will blow this cache), and if we do, having it also re-run the PHP part seems fine to me.